File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) {
        
        int odd = 0;
        int even = 0;
        foreach(int item in num_list)
            if(item % 2 == 0) 
                even++;
            else
                odd++;
        
        int[] answer = new int[2] {even, odd};
        return answer;
    }
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> num_list) {
    int odd = std::count_if(num_list.begin(), num_list.end(), [](int i){ return i & 1; });
    return vector<int>{ (int)num_list.size() - odd, odd };
}